app.rb

```diff
@@ -1,9 +1,33 @@
+require 'sinatra'
 require 'sinatra/base'
+require 'sinatra/reloader'
+require_relative 'database_connection_setup'
+require './lib/peeps'
 
 class Chitter < Sinatra::Base
+
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
     'Test page'
   end
 
+  get '/' do
+    @peeps = Peeps.all
+    erb :index
+  end
+  
+  post '/add' do
+    Peeps.create(peep: params[:peep])
+    redirect '/'
+  end
```

Comment:
If you wanted to make this more "standard", you could follow the restful routes naming conventions here: https://medium.com/@shubhangirajagrawal/the-7-restful-routes-a8e84201f206